Upgrade-Guide/op_version.md

```diff
@@ -20,3 +24,9 @@ For example, after upgrading to glusterfs-3.7.1, set op-version as:
 Note: 
 This is not mandatory, but advisable to have updated op-version if you want to make use of latest features in the updated gluster.
 
+###Client op-version
+
+To check op-version information for clients the following query can be used:
+
+    [root@~]#gluster volume status <all|VOLNAME> clients
```

Comment:
Can we add some text to explan how this helps a user? For example, get this information and the cluster max op-version information and either set it to a minimum of these 2, or upgrade the clients, or what else?